eiffel-vocabulary/EiffelAnnouncementAcknowledgedEvent.md

```diff
@@ -19,7 +19,36 @@
 The EiffelServiceDeployedEvent is to declare that an EiffelAnnouncementPublishedEvent has been received & acknowledged by a person or a machine.
 
 ## Data Members
- 
+### data.acknowledgement
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The Acknowledgement response.
+
+### data.user
+__Type:__ String  
+__Required:__ No  
+__Description:__ The identity of the user or entity that acknowledged the EiffelAnnouncementPublishedEvent. 
+
+### data.uri
+__Type:__ String  
+__Required:__ No  
+__Description:__ A URI where further information can be obtained, if applicable. 
+
+## Legal Link Types
+### CAUSE
+__Legal targets:__ Any  
+__Multiple allowed:__ Yes  
+__Description:__ Identifies a cause of the event occurring. SHOULD not be used in conjunction with __CONTEXT__: individual events providing __CAUSE__ within a larger context gives rise to ambiguity. It is instead recommended to let the root event of the context declare __CAUSE__.  
+
+### CONTEXT
+__Legal targets:__ [EiffelActivityTriggeredEvent](https://github.com/Ericsson/eiffel/blob/master/eiffel-vocabulary/EiffelActivityTriggeredEvent.md), 
```

Comment:
This means that it is impossible, in a traceable way, to acknowledge things not connected to specific activities.

Reply:
You are correct....
Suggestion to remove CONTEXT from the EiffelAnnouncementAcknowledgedEvent ?

The reason being is that the EiffelAnnouncementAcknowledgedEvent will only be used in tandem  with EiffelAnnoumncementPublishedEvent by using the CAUSE link.